attachment_synchronize/views/storage_backend_view.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+
+    <record id="view_storage_backend_form" model="ir.ui.view">
```

Comment:
Can we remove this and add a dedicated view for the task. So we can change the backend of a task easily?

Reply:
@clementmbr